core-meetings/2020-03-25-issue-management-meeting.md

```diff
@@ -0,0 +1,67 @@
+# 2020-03-25 React Native Issue Management Meeting
+
+## Agenda:
+- Issue management meeting goals
+- Workflow enhancements
+- Go through Needs: Triage issues
+- Go through Needs: Attention issues
+- Discuss areas of improvement
+- Create action items
+
+## Issue management meeting goals
+
+Héctor shared his thoughts on issue management:
```

Comment:
Do you prefer using our names or linking our usernames?

Reply:
(maybe both?)

Reply:
Done. Did I miss anybody? This was a Teams meeting, and I don't know if it is possible to look up who joined the call after the fact.